docs/sphinx_doc/user/inputs.rst

```diff
@@ -153,6 +153,16 @@ This section deals with parameters that control the simulation.
    Fixed timestep size (in seconds) used to advance the simulation. If this
    parameter is negative, then :input_param:`time.cfl` is used to determine the
    adaptive timestep during the simulation.
+   
+.. input_param:: time.initial_dt
+
+   **type:** Real number
+
+   Initial timestep size (in seconds) used to initialize the simulation. 
+   Only activated if :input_param:`time.cfl` = true and parameter is positive, 
```

Comment:
`= true` seems misleading, should just say 

```rst
if :input_param:`time.cfl` is a positive number
``` ?

Reply:
oh I meant something else, I'll fix it